app/app.js

```diff
@@ -1,22 +1,23 @@
 const express = require('express')
 const { getTopics } = require('./controllers/topics-controller')
+const { getArticleById } = require('./controllers/article-controller')
 const app = express()
 
 app.get('/api/topics', getTopics)
 
 app.get('/api', )
 
-app.use((err, request, response, next) =>{
+app.get('/api/articles/:article_id', getArticleById)
+
+app.use((err, req, res, next) =>{
     console.log(err)
-    if (err.status && err.msg) {
-        res.status(err.status).send({ msg: err.msg });
-      }
-      else if (err.code === '22P02') {
-        res.status(400).send({ msg: err.message || 'Bad Request' });
-      } else {
-        console.log(err);
+    if(err.status === 404) {
+        res.status(404).send({ msg: "Not Found" });
+    }else if (err.code === '42703') {
+        res.status(400).send({ msg: 'Bad Request' });
+    }else {
```

Comment:
You probably want to split it into two error handler, one is to handle psql errors, one is to handle custom errors